src/content/content.npmjs.js

```diff
@@ -4,18 +4,20 @@ import { mountContentScript, reloadWhenURLChanged } from './content';
 import { fetchPackageInfo } from './content-events';
 import { urlParsers } from './registry/npm';
 
+const CSS_SELECTOR_INSTALL = '.c84e15be';
+
 const addPackageReport = async (packageID) => {
   // remove an old package report (if exists)
   const currPackageReport = document.getElementsByTagName(packageReportTagName);
   if (currPackageReport?.length) {
     currPackageReport.item(0).remove();
   }
 
-  const repository = await waitForElement('#repository', document.querySelector('#main'));
+  const install = await waitForElement(CSS_SELECTOR_INSTALL, document.querySelector('#main'));
   const packageReport = document.createElement(packageReportTagName);
   packageReport.setAttribute('package-type', packageID.type);
   packageReport.setAttribute('package-name', packageID.name);
-  repository.parentElement.insertBefore(packageReport, repository);
+  install.parentElement.insertBefore(packageReport, install);
```

Comment:
Did you changed the location of where Overlay is displayed?

If so, please attach a screenshot of a page with the new location.

Reply:
yes. the change it tiny.

![Untitled1](https://github.com/os-scar/overlay/assets/44925669/90d17cb8-e48e-46ea-a47b-cfb95f05cafb)